example/ios/WaasSdkReactNativeExample.xcodeproj/project.pbxproj

```diff
@@ -495,7 +597,57 @@
 				CLANG_ENABLE_MODULES = YES;
 				CURRENT_PROJECT_VERSION = 1;
 				ENABLE_BITCODE = NO;
+				HEADER_SEARCH_PATHS = (
+					"$(inherited)",
+					"/opt/homebrew/opt/openssl@1.1/include",
```

Comment:
I couldn't get SSL to install locally when developing. The only thing that worked for me was using `brew install openssl@1.1` and pointing to the headers locally.